modules/cnf-configuring-amq-interconnect-messaging-bus.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-configuring-amq-interconnect-messaging-bus_{context}"]
+= Configuring the AMQ messaging bus for PTP fast event notifications
+
+To pass fast event notifications between publisher and subscriber on a node, you must install and configure an AMQ messaging bus to run locally on the node. You do this by installing the AMQ Interconnect Operator for use in the cluster and configuring it.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the AMQ Interconnect Operator.
+
+.Procedure
+
+. Check that the AMQ Interconnect Operator was installed correctly and the required pods are running. By default, the AMQ Interconnect Operator installs to the `cloud-native-events` namespace:
++
+[source,terminal]
+----
+$ oc get pods -n cloud-native-events
+----
++
+.Example output
```

Comment:
These are example consumer pods , do we need to document them?
I think this should be pods under openshift-ptp namespace (producers)